java-client/flight/src/main/java/io/deephaven/client/impl/FieldVectorAdapter.java

```diff
@@ -204,80 +283,234 @@ public FieldVector visit(DoubleArray doubleArray) {
         return vector;
     }
 
-    FieldVector visitBooleanArray(GenericArray<Boolean> array) {
+    FieldVector visitBooleanElements(Collection<Boolean> elements) {
```

Comment:
It looks like we could make these methods `private`. Is there benefit to leaving them package-private?